src/controllers/routes.js

```diff
@@ -0,0 +1,23 @@
+'use strict'
+
+const requestUtil = require('five-bells-shared/utils/request')
+const routingTables = require('../services/routing-tables')
+const routeBroadcaster = require('../services/route-broadcaster')
+const knownConnectors = {}
+
+exports.post = function * () {
+  const routes = yield requestUtil.validateBody(this, 'Routes')
+
+  // TODO verify that POSTer of these routes matches route.connector.
```

Comment:
@sentientwaffle @justmoon How are connectors going to verify routes are coming from the right connector?